# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/SheetLoadData.h
# Commit: baa464d5f255
# Full Hash: baa464d5f25599d4644ac371593d8cfec300d349
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2024-10-02 09:50:09
# Regressor Bug: 1899734
# File Overlap Count: 1
# Description:
#   Bug 1899734 - Part 4: Add SheetLoadData::InitiatorTypeString method. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223350
# ==============================================================================

diff -r 432c1fa3fbcc -r baa464d5f255 layout/style/SheetLoadData.h
--- a/layout/style/SheetLoadData.h	Tue Oct 01 12:17:20 2024 +0000
+++ b/layout/style/SheetLoadData.h	Tue Oct 01 12:17:21 2024 +0000
@@ -301,6 +301,8 @@
     mExpirationTime.SetMinimum(aExpirationTime);
   }
 
+  nsLiteralString InitiatorTypeString();
+
  private:
   const SheetLoadData& RootLoadData() const {
     const auto* top = this;
